crmsh/report/collect.py

```diff
@@ -65,9 +65,15 @@ def collect_ha_logs(context: core.Context) -> None:
     Collect pacemaker, corosync and extra logs
     """
     log_list = [get_pcmk_log(), get_corosync_log()] + context.extra_log_list
-    for log in log_list:
+    log_list = [
+        os.path.expanduser(log)
+        if log.startswith("~") else log
```

Comment:
No need to test `startwith("~")`. `os.path.expanduser()` handles paths without leading `~` correctly.